Automattic/wp-calypso
# ActivityLog: Add original publish date

Related to p1HpG7-oTm-p2

## Proposed Changes

* Add an `Published: ` to post item types in the Activity Log

## Todo

* Update / fix tests for links

## Testing Instructions



* If D123264 hasn't landed on wpcom you'll need to apply that to a WordPress.com sandbox and sandbox your public API as described in that patch.
* Launch this branch locally or in a live container and visit a site's activity log which has some published posts.
* Ensure the published posts have an `Published: ` present on the Activity Card
* Posts that have yet to be published should show nothing for this.
* Posts of other types, such as updated/modified for an already published post should also show the original publish date.
* If there are modified / update events prior to the publish date, it is expected that those will show nothing, as the Activity Log isn't retroactive.
* These cases should be true in both WordPress.com and Jetpack Cloud

<img width="698" alt="Screenshot 2023-09-26 at 5 25 53 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/da16dc71-cfa7-4412-91e7-d4ca89d563f6">
<img width="1056" alt="Screenshot 2023-09-26 at 5 26 13 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/51656e97-d27f-49be-b8ff-0ab7e42eabeb">


## Pre-merge Checklist
